### ISSUE TITLE
Provide some vumeters (volume indicators) in the input choices widgets?

### ISSUE THREAD

**Issue by nekohayo** [ryonakano/reco#236]
This enhancement suggestion is not mission-critical, but would be pretty convenient to ensure users don't make mistakes (or to reassure them) before starting recording.

---

In issue #179, a live scrolling peakmeter histogram was implemented while recording happens.

However, there is no easy way to know which device is the correct device _before_ hitting record, and also when you're mixing the system + microphone inputs, you don't get a distinction between the two in the peakmeter, so it can be possible sometimes that you're effectively recording only the system's monitor (output) and the input is silent because you're recording the wrong input.

In this microphone choice widget:

![image](https://github.com/ryonakano/reco/assets/479401/a4028b9f-7264-4c70-888c-834eb7b87d0a)

...it could be nice to have some basic vumeter that can help the user identify which devices are making sound or not. Maybe a simple GTK LevelBar? Or something like what GNOME Settings shows on the right (maybe a bit less narrow, though) : 

![image](https://github.com/ryonakano/reco/assets/479401/fba56db7-bc2a-4baf-93fb-50ac673ba1fa)
